DOCS/xml-fields.md

```diff
@@ -92,14 +90,13 @@ Example:
 
 ### Section annotation markers
 
-Some portions of the license text can be surrounded by annotation tags, to denote certain sections of the license text:
+Some portions of text can be surrounded by annotation tags. These are all optional:
 
-* **`<titleText>`**: indicates the title of the license, if specified in the license text itself
-* **`<copyrightText>`**: indicates where a copyright notice would be placed for the licensed code
+* **`<titleText>`**: indicates the title of the license, if specified in the license text itself. If the license does not have a title, then this tag is not needed.
+* **`<copyrightText>`**: indicates where a copyright notice would be placed for the licensed code. Because the copyright notice is not part of the license and thus ignored as related to matching a license, it does not matter if the the copyright notice includes a specific name or is generic.
   * Note that `<copyrightText>` should _not_ be used for copyright notices that apply to the copyright in the license text itself!
-* **`<standardLicenseHeader>`**: indicates a standard way that the license recommends specifying the license, e.g. in code comments
-  * Note that `<standardLicenseHeader>` should _only_ be used if it is defined by the license text itself!
-  * Typically, this would be located in an appendix titled something like "How to apply the license to your work"
+* **`<standardLicenseHeader>`**: indicates a standard way that the license recommends specifying the license, see license-fields:standard-license-header](license-fields.md#h-standard-license-header) for a full descriptioin of this field.
```

Comment:
missing square bracket before license-fields, so the link isn't rendering correctly